Makefile

```diff
@@ -305,13 +312,16 @@ DEF_INC_CFLAGS := $(foreach i,$(INCLUDE_DIRS),-I$(i)) $(C_DEFINES)
 ifneq (,$(call find-command,clang))
   CPP      := clang
   CPPFLAGS := -E -P -x c -Wno-trigraphs $(DEF_INC_CFLAGS)
-else
+else ifneq (,$(call find-command,cpp))
   CPP      := cpp
   CPPFLAGS := -P -Wno-trigraphs $(DEF_INC_CFLAGS)
+else
+  $(error Unable to find cpp or clang)
 endif
 
 # Check code syntax with host compiler
-CC_CHECK := gcc
+# detect gcc
```

Comment:
Remove this comment. If you want to make it more clear that this is a terminal command. You could change 
```
# Use gcc for checking code syntax which does a better job than ido.
```

Reply:
useless so I just remove it